rts/lra/test/basic/src/test/java/io/narayana/lra/arquillian/client/NarayanaLRAClientIT.java

```diff
@@ -52,8 +52,12 @@ public void testGetAllLRAs() {
         URI lra = lraClient.startLRA("test-lra");
 
         List<LRAData> allLRAs = lraClient.getAllLRAs();
-
         Assert.assertTrue(allLRAs.stream().anyMatch(lraData -> lraData.getLraId().equals(lra.toASCIIString())));
+
+        lraClient.closeLRA(lra);
```

Comment:
Would not be good to put this under `finally` block?

Reply:
I don't think there is a need. We don't do that in other tests AFAICT. We expect that the LRA coordinator is restarted (thus LRAs are cleared) between runs. Maybe we can go through all tests and fix this, but personally, I don't think that it is necessary.